src/px-script/main/topbar/NavMenu.js

```diff
@@ -1,11 +1,16 @@
 import { isWithinBoundingBox } from "../utils";
 
+const SELECTORS = {
+    BTN: ".topbar-btn",
+    ICON: "topbar-btn-icon"
+};
+
 export default class NavMenu {
     constructor (topbarComponent, navMenu) {
         this.navMenuElement = navMenu;
         this.isOpen = false;
-        this.btnElement = topbarComponent.querySelector(".topbar-btn");
-        this.iconElement = this.btnElement.querySelector(".topbar-btn-icon");
+        this.btnElement = topbarComponent.querySelector(SELECTORS.BTN);
+        this.iconElement = this.btnElement.querySelector(SELECTORS.ICON);
```

Comment:
Can `this.btnElement` be `null`?

Reply:
Yes; added ternary to `this.iconElement` declaration.